hardware_interface/include/hardware_interface/joint_handle.hpp

```diff
@@ -15,21 +15,16 @@
 #ifndef HARDWARE_INTERFACE__JOINT_HANDLE_HPP_
 #define HARDWARE_INTERFACE__JOINT_HANDLE_HPP_
 
-#include <string>
-
 #include "hardware_interface/handle.hpp"
-#include "hardware_interface/macros.hpp"
-#include "hardware_interface/visibility_control.h"
 
 namespace hardware_interface
 {
 /** A handle used to get and set a value on a given joint interface. */
-class JointHandle : public Handle<JointHandle>
+class JointHandle : public ReadWriteHandle<JointHandle>
```

Comment:
Why not use `CommandHandle`

Reply:
Or is this also part of the transmissions?

Reply:
This PR is a wee bit misleading in the title and description. Not only component interfaces were remodeled but the existing ones were also touched up to avoid code duplication.